draft-rha-jose-hpke-encrypt.md

```diff
@@ -199,6 +200,14 @@ The recipient will create the receiving HPKE context by invoking SetupBaseR() (S
 
 The Open function will, if successful, decrypts "ct".  When decrypted, the result will be either the CEK (when Key Encryption mode is used), or the content (if Integrated Encryption mode is used).  The CEK is the symmetric key used to decrypt the ciphertext.
 
+## Protected Header Parameters
+
+In order to ensure the security properties of HPKE, the following requirements MUST be followed for protected header parameters:
+
+* For Integrated Encryption, "epk" MUST be present, and MUST contain an Encapsulated JSON Web Key.
```

Comment:
This needs to go the according mode section.

Reply:
Agreed, moved text.